packages/teams-js/src/public/pages.ts

```diff
@@ -292,13 +292,12 @@ export namespace pages {
   }
 
   /**
-   * Navigate to the given application ID and page ID, with optional parameters for a WebURL (if the application
-   * cannot be navigated to, such as if it is not installed), Channel ID (for applications installed as a channel tab),
-   * and sub-page ID (for navigating to specific content within the page). This is equivalent to navigating to
-   * a deep link with the above data, but does not require the application to build a URL or worry about different
-   * deep link formats for different hosts.
-   * @param params - Parameters for the navigation
-   * @returns a promise that will resolve if the navigation was successful
+   * Used to navigate to apps other than your own.
+   *
+   * If you are looking to navigate within your own app you should prefer {@link pages.currentApp.navigateToDefaultPage} or {@link pages.currentApp.navigateTo}
```

Comment:
I would usually use _can_, _might_, or _must_ vs _should_. When you use _should_ or _may_, there can be ambiguity on what's a correct/required way to do something and what's just a suggestion. If it is just a suggestion, than maybe instead of prefer use consider "If you want to navigate within your own app, you should consider using the strongly-typed functions pages.currentApp.navigateToDefaultPage() or pages.currentApp.navigateTo().